nixos/modules/system/boot/systemd.nix

```diff
@@ -112,6 +114,7 @@ let
       "sleep.target"
       "hybrid-sleep.target"
       "systemd-hibernate.service"
+      "systemd-hibernate-clear.service"
```

Comment:
From the release notes:

> A tiny new service `systemd-hibernate-clear.service` has been added which clears hibernation information from the HibernateLocation EFI variable, in case the resume device is gone. Normally, this variable is supposed to be cleaned up by the code that initiates the resume from hibernation image. But when the device is missing and that code doesn't run, this service will now do the necessary work, ensuring that no outdated hibernation image information remains on subsequent boots.

See also https://www.freedesktop.org/software/systemd/man/latest/systemd-hibernate-resume.service.html